docs/dev/developer-guides/building-on-zksync/events.md

```diff
@@ -14,30 +14,50 @@ At zkSync, events behave the same way as in Ethereum.
 ## Events filtering
 
 Filtering is used to query indexed data and provide lower-cost data storage when the data is not required to be accessed on-chain.
+While filtering, we advice to load events by block ranges (0-1999, 2000-3999, ...) and index the result on your end, else you will get an error that says "block range should be less than or equal to 2000".
 
 These can be used in conjunction with the [Provider Events API](https://docs.ethers.io/v5/api/providers/provider/#Provider--event-methods) and with the [Contract Events API](https://docs.ethers.io/v5/api/contract/contract/#Contract--events).
 
+## Getting the events
 
+To get the events, use the following template:
 
-```solidity
-    abi = [
-  "event Transfer(address indexed src, address indexed dst, uint val)"
-];
+```js
+const ether = require("ethers");
+const contractABI = require("./ABI_JSON")
 
-contract = new Contract(tokenAddress, abi, provider);
+const getEvent = async () => {
+const contractAddress = "<YOUR ADDRESS>"
```

Comment:
< CONTRACT ADDRESS >

Reply:
fixed in 79e18fb8c9dd426d700ab809eaeff8251ccb799e